### DISCUSSION TITLE
盲审版本所有论文不再展示，如何设置？

### DISCUSSION THREAD

**Discussion by WendongZh** [sjtug/SJTUThesis#907] [提问] [answered]
改成盲审版本之后，之前在achievement.tex文件中\begin{bibliolist} .. \end{bibliolist}内的论文信息全部没有了，实际上学校要求只把作者删除就行，这里需要怎么更改？

**Comment by AlexaraWu** [comment, marked as answer]
仔细看一下模板给的示例。正常情况下展示 `bibliolist` 环境里的内容，盲审模式下展示 `bibliolist*` 环境里的内容。

**Reply by WendongZh** [reply]
哈哈哈，我改成bibliolist*之后会报错，显示“Something's wrong--perhaps a missing \item. \end{bibliolist*}”

比如改成
\begin{bibliolist*}
	\item 第一作者。xxxx
\end{bibliolist*}

**Reply by AlexaraWu** [reply]
少个参数。

```latex
\begin{bibliolist*}{00}
  \item 第一作者。xxxx
\end{bibliolist*}
```

**Reply by WendongZh** [reply]
哦哦，好的，知道了。哈哈哈，这也行。。。

**Reply by WendongZh** [reply]
多谢多谢，我tm搞了半天。。。笑死
